yb_migrate/src/migration/oracle.go

```diff
@@ -282,15 +279,15 @@ func getSourceDSN(source *utils.Source) string {
 		os.Exit(1)
 	}
 
+	log.Infof("Source DSN used for export:%s\n", sourceDSN)
 	return sourceDSN
 }
 
 func OracleGetAllTableNames(source *utils.Source) []string {
 	dbConnStr := GetDriverConnStr(source)
```

Comment:
We can log the golang connection string here and other similar places.

Reply:
This will be handled in the logging for common.go, that is where the connection string is formed